types/ssh2-sftp-client/index.d.ts

```diff
@@ -4,16 +4,16 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 import * as ssh2 from 'ssh2';
-
+import * as ssh2Stream from 'ssh2-streams';
 export = sftp;
 
 declare class sftp {
   connect(options: ssh2.ConnectConfig): Promise<void>;
   list(remoteFilePath: string): Promise<sftp.FileInfo[]>;
   get(remoteFilePath: string, useCompression?: boolean, encoding?: string | null): Promise<NodeJS.ReadableStream>;
-  fastGet(remoteFilePath: string, localPath: string, options?: any): Promise<any>;
+  fastGet(remoteFilePath: string, localPath: string, options?: ssh2Stream.TransferOptions): Promise<any>;
```

Comment:
Does `ssh2-streams` have also a type for the returned value of the `Promise`?

Reply:
From [the implementation](https://github.com/jyu213/ssh2-sftp-client/pull/64/files), it looks like we should type this as a `Promise<string>`.

Reply:
Yep, can you change it?

Reply:
Great catch !! Made the changes :smile: